cmake/k4FWCoreCreateConfig.cmake

```diff
@@ -0,0 +1,6 @@
+include(CMakePackageConfigHelpers)
+
+configure_package_config_file(${PROJECT_SOURCE_DIR}/cmake/k4FWCoreConfig.cmake.in
+                              ${PROJECT_BINARY_DIR}/k4FWCoreConfig.cmake
+                              INSTALL_DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/k4FWCore
+                              PATH_VARS CMAKE_INSTALL_INCLUDEDIR CMAKE_INSTALL_LIBDIR)
```

Comment:
I would put these in the main `CMakeLists.txt`, it's only a few more lines.

Reply:
It will become a few more once we start to also export version information properly. Since we have a usable tag already, should I add that as well?

Reply:
Yeah, why not